primitives/Title.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { ReactNode } from 'react';
```

Comment:
```suggestion
import React, { type ReactNode } from 'react';
```

Explicitly stating that the import is a `type`